packages/arb-token-bridge-ui/src/state/app/utils.ts

```diff
@@ -67,7 +67,7 @@ export const transformDeposits = (
         ? getStandardizedTimestamp(tx.timestampResolved)
         : null,
       txId: tx.txID,
-      asset: tx.assetName?.toLowerCase() || '',
+      asset: tx.assetName || '',
```

Comment:
This change removes asset names being force lower-cased coming from subgraph/event logs.